setup.py

```diff
@@ -1 +0,0 @@
-__import__("setuptools").setup()
```

Comment:
Do not delete this file

Reply:
Why not?

Reply:
The move to pyproject.toml was done in https://github.com/mgaitan/sphinxcontrib-mermaid/pull/154, this is the correct setup